packages/pds-ember/addon/components/pds/external-link/index.hbs

```diff
@@ -0,0 +1,15 @@
+<a
+  class="pds--external"
+  rel="noopener noreferrer"
+  target="_blank"
+  ...attributes
+  data-test-external-link
+>
+  {{yield}}
+  {{#if this.showIcon}}
```

Comment:
using a getter instead of `(not @hideIcon)`, to avoid dependency on 3rd-party helper

Reply:
`unless` is built-in if you want to use that

Reply:
(facepalm) I completely forgot about `unless`. That might do the trick.